src/components/sequencing/SequenceEditor.svelte

```diff
@@ -274,6 +279,77 @@
     }
   }
 
+  const blockMark = Decoration.mark({ class: 'cm-block-match' });
+
+  const blockTheme = EditorView.baseTheme({
+    '.cm-block-match': {
+      outline: '1px dashed',
+    },
+  });
+
+  function highlightBlock(viewUpdate: ViewUpdate) {
```

Comment:
This should move to language specific code, but deferring to minimize merge headache